### PR TITLE
Add JupyterLab UX survey from JupyterCon 2018

### PR DESCRIPTION
**Pull request by Zsailer**
This PR includes user-testing data collected at JupyterCon 2018.

### CODE REVIEW

**Comment by Zsailer** [comment]
@tgeorgeux I think this survey is ready for merge. We can add new analysis later.

File: surveys/2018-09-jupyterlab-ux/README.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+# JupyterLab UX Survey
+
+This folder contains user-testing data collected at JupyterCon 2018 (August 2018) by Cal Poly Interns and Tim George (@tgeorgeux).
+
+The goal of the testing was to answer specific questions about JupyterLab UX. See the [analysis notebook](analysis/questions-to-answer.ipynb) for details.
+
+**How to run the notebook**
+
+
```
Comment (parente) [inline surveys/2018-09-jupyterlab-ux/README.md]: I was planning on sending a PR this evening in response to a request in the Jupyter Google Group to make this entire repo runnable as a Binder. I think the libs used in the other surveys and this one likely overlap and are the usual culprits.

**Comment by parente** [comment]
Awesome to see more survey data here!

File: surveys/2018-09-jupyterlab-ux/README.md (hunk 1 above)
Comment (Zsailer) [reply surveys/2018-09-jupyterlab-ux/README.md]: A Binder for this repo would be awesome! It looks like PR #5 includes all the requirements necessary for this survey. Should I remove the requirements/install instructions from this directory?

**Comment by Zsailer** [comment]
I just pushed a few more graphs to the notebook. 

I think @tgeorgeux is writing up a post regarding this survey.

File: surveys/2018-09-jupyterlab-ux/README.md (hunk 1 above)
Comment (parente) [reply surveys/2018-09-jupyterlab-ux/README.md]: I see no harm in having both. I think of the binder one as a convenience but the requirements listed in each survey subdir as the source of truth.

**Comment by Zsailer** [comment]
@tgeorgeux and I added one more set of analyses from JupyterCon 2018!

**Comment by tgeorgeux** [comment]
@ellisonbg do you mind merging this?

**Pull request merged**